Documentation/conf.py

```diff
@@ -47,6 +47,10 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates']
 
+locale_dirs = ['locale/']
+gettext_uuid = True
+gettext_compact = False
```

Comment:
What are these for?

Reply:
Just following the RTD documentation, maybe I should also link this in the PR commit/description:

https://docs.readthedocs.io/en/stable/guides/manage-translations-sphinx.html